ColumnSet.js

```diff
@@ -182,6 +182,19 @@ define([
 			this.on('.dgrid-column-set:dgrid-cellfocusin', function (event) {
 				self._onColumnSetCellFocus(event, this);
 			});
+
+			if ('expand' in this) {
+				aspect.after(this, 'expand', this.afterRowExpand, true);
+			}
+		},
+
+		afterRowExpand: function (row, expand) {
```

Comment:
It occurs to me now (apparently didn't where I originally fixed this in another project, which always used sync stores) that this should probably be waiting for the promise returned from `expand` to resolve.  We should hook this up without `receiveArguments` set so that we get the returned promise here (and don't forget to return it again).